qubes-menus/menus/xfce-qubes-applications.menu

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE Menu PUBLIC "-//freedesktop//DTD Menu 1.0//EN"
+  "http://www.freedesktop.org/standards/menu-spec/1.0/menu.dtd">
+
+<Menu>
```

Comment:
A menu file in a common package should have a common name - something like `qubes-applications.menu`. Then, individual desktop environment packages can either use it directly, create a symlink, or use a different file (least preferred).

Reply:
At least this is the case for KDE, the menu is not the same. So I would use xfce-qubes-applications.menu as qubes-applications.menu